README.md

```diff
@@ -100,19 +114,27 @@ class SampleForm extends React.Component {
 
 ## Documentation
 
-Visit [docs](https://github.com/mbasso/react-decoration/blob/master/docs) folder to find the complete list of decorators and their usage.
+Visit
+[docs](https://github.com/mbasso/react-decoration/blob/master/docs)
+folder to find the complete list of decorators and their usage.
 
 ## Change Log
 
-This project adheres to [Semantic Versioning](http://semver.org/).  
-Every release, along with the migration instructions, is documented on the Github [Releases](https://github.com/mbasso/react-decoration/releases) page.
+This project adheres to [Semantic Versioning](http://semver.org/).
+Every upstream release, along with the migration instructions, is
+documented on the Github
+[Releases](https://github.com/mbasso/react-decoration/releases) page.
 
 ## Authors
+
 **Matteo Basso**
 - [github/mbasso](https://github.com/mbasso)
 - [@Teo_Basso](https://twitter.com/Teo_Basso)
+- Ashley Lake <ashley.lake@regence.com
```

Comment:
Can you please create a new entry with your name and an item that is the email?
I think that in this way it might be confused as one of my contacts, like GitHub or twitter. As a new list item it will be also more recognizable